File: backend/src/main/java/com/common/nayong/enumerate/PaymentType.java
package com.common.nayong.enumerate;

public enum PaymentType {
    creditCard(1, "credit_card"),
    paypal(2, "paypal"),
    applePay(3, "apple_pay"),
    googlePay(4, "google_pay");

    public final int key;
    public final String name;

    PaymentType(int key, String name) {
        this.key = key;
        this.name = name;
    }

    public String toString() { return name; }
}

File: frontend/src/script/enums/paymentType.js
'use strict'

export const creditCard = "credit_card"
export const paypal = 'paypal'
export const applePay = 'apple_pay'
export const googlePay = 'google_pay'

export const paymentList = [
	{ key: 1, name: "credit_card", display: "Credit Card", picUrl: new URL('../../assets/credit_card.png', import.meta.url).href },
	{ key: 2, name: "paypal", display: "PayPal", picUrl: new URL('../../assets/paypal.png', import.meta.url).href },
	{ key: 3, name: "apple_pay", display: "Apple Pay", picUrl: new URL('../../assets/apple_pay.png', import.meta.url).href },
	// { key: 4, name: "google_pay", display: "Google Pay", picUrl: new URL('../../assets/google_pay.png', import.meta.url).href }
]